### PR TITLE
round image corners

### PR DESCRIPTION
**Pull request by alexfriesen**
This PR adds rounded corners to feature, showcase and news images.

### CODE REVIEW

File: themes/godotengine/pages/features.htm [hunk 1]
```diff
@@ -56,18 +58,18 @@ <h2>Innovative design</h2>
...
 	  </div>
 	</div>
-  </section>
+</section>
 
-  <section id="features_3d" class="container sm-full">
+<section id="features_3d" class="container sm-full">
 	<a href="{{ 'assets/features/3dgames.jpg' | theme }}" class="lightbox" data-group="features">
 	  <img
-		src="{{ 'assets/features/3dgames.jpg' | theme }}"
```
Comment (coppolaemilio) [inline themes/godotengine/pages/features.htm:64]: It seems like you are changing from tabs to spaces for indenting in some areas. I don't have any strong preference for either, but I'm not sure if the project has some indentation rules set up by the linter. Maybe @Calinou or @YuriSizov can clear it up?

Comment (alexfriesen) [reply themes/godotengine/pages/features.htm:64]: oops, fixed that

**Comment by coppolaemilio** [comment]
The aspect ratio of the videos changed because of the `.showcase-card` property `height: 16rem;` 
This should not be the case in the showreel videos:

https://user-images.githubusercontent.com/2206700/192314416-753bf0b1-884f-4976-a133-e214ff2f8e67.mp4

**Comment by alexfriesen** [comment]
@coppolaemilio I must have overlooked that one. Videos should look proper again

**Comment by coppolaemilio** [comment]
yeah, looks good now! thanks for all your recent help with the site :)

**Pull request merged**